testutil-mentions/src/main/kotlin/io/spine/examples/pingh/testing/mentions/given/PredefinedGitHubResponses.kt

```diff
@@ -69,13 +70,21 @@ public class PredefinedGitHubResponses : GitHubClientService {
      */
     private var responseStatusCode = HttpStatusCode.OK
 
+    /**
+     * List of values of the `updatedAfter` arguments which were passed to [fetchMentions] methods.
+     *
+     * The order of the elements corresponds to the order of methods call.
+     */
+    private val updatedAfterList = mutableListOf<Timestamp>()
```

Comment:
Also, in a vacuum, `updatedAfterList` is weird by itself. I would name it better, but I need to understand the idea first.